pygmt/tests/test_clib.py

```diff
@@ -743,6 +743,23 @@ def test_dataarray_to_matrix_inc_fails():
         dataarray_to_matrix(grid)
 
 
+def test_dataarray_to_matrix_zero_inc_fails():
+    """
+    Check that dataarray_to_matrix fails for zero increments grid.
+    """
+    data = np.ones((5, 5), dtype="float64")
+    x = np.linspace(0, 1, 5)
+    y = np.zeros_like(x)
+    grid = xr.DataArray(data, coords=[("y", y), ("x", x)])
+    with pytest.raises(GMTInvalidInput):
+        dataarray_to_matrix(grid)
+    y = np.linspace(0, 1, 5)
```

Comment:
Add a blank line separating these two sub-tests.
```suggestion
        dataarray_to_matrix(grid)

    y = np.linspace(0, 1, 5)
```